app.py

```diff
@@ -0,0 +1,123 @@
+from cs50 import SQL
+from flask import Flask, flash, redirect, render_template, request, session
+from datetime import datetime
+from flask_session import Session
+from tempfile import mkdtemp
+from werkzeug.security import check_password_hash, generate_password_hash
+
+from helpers import apology, login_required
+
+# Configure application
+app = Flask(__name__)
+
+# Ensure templates are auto-reloaded
+app.config["TEMPLATES_AUTO_RELOAD"] = True
+
+# Configure session to use filesystem (instead of signed cookies)
+app.config["SESSION_PERMANENT"] = False
+app.config["SESSION_TYPE"] = "filesystem"
+Session(app)
+
+# Configure CS50 Library to use SQLite database
+db = SQL("sqlite:///rockets.db")
+
+# I don't understand if this is needed. Copying from /finance
```

Comment:
This pretty useful for  testing, it should mean that when the client requests a page, that it doesn't assume it already has the most recent version locally.